lib/sul_styles/engine.rb

```diff
@@ -0,0 +1,11 @@
+module SulStyles
+  class Engine < ::Rails::Engine
+    isolate_namespace SulStyles
+
+    config.after_initialize do |app|
+      unless Rails.env.production?
```

Comment:
@cbeer this is something that @mejackreed and I were talking about the other day, which is that SULStyles could provide a style guide page that devs could look at as a pallet for development and SUL Styles could use for to do automated testing against. I think we assumed that this style-guide page shouldn't necessarily be available in your app when running in production mode (similar to the teaspoon testing page).